chess/xboard.py

```diff
@@ -572,14 +572,14 @@ def draw(self, async_callback=None):
         command = self.command("draw")
         return self._queue_command(command, async_callback)
 
-    def pondering(self, ponder, async_callback=None):
+    def pondering(self, ponder=False, async_callback=None):
```

Comment:
I'm not so sure about this because this default argument will imply that when someone calls `pondering()` it will in fact be turned off which seems un-intuitive. If anything, it should be `True`. In fact now that I think about it it should probably be renamed to `ponder()` since it's not the actual Xboard spec (which has `hard` and `easy`)